### PR TITLE
Create Add-music.md

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Added the music task

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
@niccokunzmann

File: tasks/Add-music.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Hints:
+- If someone wants to work with you, let them do that. You can use techniques like [pair programming](https://www.youtube.com/watch?v=vgkahOzFH2Q) or collaborate together on branches of your forks. You can attribute a pull-request also to other people who helped you out or submit a pull-request where you and an other person contributed together.
+- You can join our [chat on gitter](https://gitter.im/fossasia/labyrinth) if you get stuck to get help or just hang out and have a nice time with others.
+
+---
+
+Categories:
+- [X] Code
+- [X] User Interface
```
Comment (duttaditya18) [inline tasks/Add-music.md]: I don't think it is really related to UI.

File: tasks/Add-music.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+To make the game more interesting, please add a functionality to add music when the user enters a tile,
+
+Requirements:
+
+- Add the music with the relevant code changes to the [tiles.js](https://github.com/fossasia/labyrinth/blob/master/js/tiles.js) file.
+- It should automatically play when the users enters, and stop when exiting. 
+- It should be music relevant to the theme, try not to add any songs
+
+Expected Outcome:
+- a merged pull request
```
Comment (duttaditya18) [inline tasks/Add-music.md]: Capitalization.

File: tasks/Add-music.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Labyrinth: Add background music to tiles
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To make the game more interesting, please add a functionality to add music when the user enters a tile,
+
+Requirements:
+
+- Add the music with the relevant code changes to the [tiles.js](https://github.com/fossasia/labyrinth/blob/master/js/tiles.js) file.
```
Comment (duttaditya18) [inline tasks/Add-music.md]: Tell the users to use the API. (That I am working on.)

**Comment by niccokunzmann** [comment]
Thanks!
I think, if we add a note about the law & music rights & self-made music in the future, this will make it clearer what can be accepted.

**Comment by arjunpandey2003** [comment]
sure making updates

**Pull request merged**